apis/cloud.redhat.com/v1alpha1/clowdapp_types.go

```diff
@@ -50,6 +53,9 @@ type DatabaseSpec struct {
 	// name of the logical database inside the database server in (*_local_*) mode
 	// and the name of the secret to be used for Database configuration in (*_app-interface_*) mode.
 	Name string `json:"name,omitempty"`
+
+	// Defines the Name of the app to share a database from
+	SharedDBAppName string `json:"sharedDbAppName,omitempty"`
```

Comment:
Have you considered defining `shared bool` instead of `SharedDBAppName string`?  We'd then rely on `Name` for both the shared and non-shared DB name.

Reply:
I did, but this would be overloading the term as we look for the "app" name to share from, this insulates the dependent app from a dbname change, as they ask for the app, not the db.